README.md

```diff
@@ -174,3 +174,25 @@ Add a role that can be used by cloudcheckr to collect data and stats
 
 
 ```
+
+
+## CloudWatch Monitoring role
+Adding role for cloudwatch monitoring to allow instance to send custom metrics
+
+### Available variables
+* [`instance_role`]: String: The name of the instance role to attach the policies to.
```

Comment:
Can you update with whether variables are required or optional? See https://github.com/skyscrapers/terraform-network/blob/master/README.md for an example